core/src/test/java/com/scalar/db/storage/cassandra/CassandraAdminTest.java

```diff
@@ -58,8 +58,8 @@ public void setUp() throws Exception {
   @Test
   public void getTableMetadata_ClusterManagerShouldBeCalledProperly() throws ExecutionException {
     // Arrange
-    String namespace = "ns";
-    String table = "table";
+    String namespace = "sample_ns";
```

Comment:
Just a question. Why do we need rename it?

Reply:
The `table` is a reserved keyword, if we keep using it then the expected query has to be changed as well. So I renamed it.

Reply:
So we need to add other unit tests to check if all namespaces/tables/columns with reserved keywords are quoted.

Reply:
I added the unit tests and the integration test as well for reserved keywords. Please take a look again.